### PR TITLE
Sync slave interactions and prisoner target ideology

### PR DESCRIPTION
**Pull request by SokyranTheDragon**
This is NOT related to #213 and does not fix it.

### CODE REVIEW

**Comment by SokyranTheDragon** [comment]
Originally wanted to make this into 2 PRs, but since they both ended up working on the same file, and ended up needing the same SyncWorker, I decided to go with a single one.

Though, I suppose I could have used SyncField instead.

File: Source/Client/Sync/SyncDictionary.cs [hunk 1]
```diff
@@ -525,6 +525,11 @@ public static class SyncDictionary
                 (ByteWriter data, ITab_ContentsTransporter tab) => { },
                 (ByteReader data) => new ITab_ContentsTransporter()
             },
+            {
```
Comment (Zetrith) [inline Source/Client/Sync/SyncDictionary.cs:528]: These should be split into specific workers for vanilla types.

File: Source/Client/Sync/Game/SyncFields.cs [hunk 2]
```diff
@@ -231,6 +233,7 @@ static void ITab_Pawn_Visitor(ITab __instance)
             Pawn pawn = __instance.SelPawn;
             SyncMedCare.Watch(pawn);
             SyncInteractionMode.Watch(pawn);
+            SyncSlaveInteractionMode.Watch(pawn);
```
Comment (Zetrith) [inline Source/Client/Sync/Game/SyncFields.cs:236]: ideoForConversion also could be synced here to fix #213

File: Source/Client/Sync/SyncDictionary.cs (hunk 1 above)
Comment (SokyranTheDragon) [reply Source/Client/Sync/SyncDictionary.cs:528]: Honestly, I originally tried that but it didn't work... I guess I made some mistake that I must have missed. All is fine now.

File: Source/Client/Sync/Game/SyncFields.cs (hunk 2 above)
Comment (SokyranTheDragon) [reply Source/Client/Sync/Game/SyncFields.cs:238]: Huh. Haven't realized it would be that easy to fix.

**Pull request merged**